Content.Client/StationRecords/GeneralStationRecordConsoleWindow.xaml

```diff
@@ -1,17 +1,35 @@
 <DefaultWindow xmlns="https://spacestation14.io"
                Title="{Loc 'general-station-record-console-window-title'}"
                MinSize="750 500">
-    <BoxContainer>
-        <!-- Record listing -->
-        <BoxContainer Orientation="Vertical" Margin="5 5 5 5" MinWidth="250" VerticalExpand="True">
-            <Label Name="RecordListingStatus" Visible="False" />
-            <ScrollContainer VerticalExpand="True">
-                <ItemList Name="RecordListing" />
-            </ScrollContainer>
+    <BoxContainer Orientation="Vertical">
+        <BoxContainer Orientation="Vertical" Margin="5 10 5 5"
+                HorizontalExpand="true">
+            <BoxContainer Margin="0 0 0 5" HorizontalExpand="true">
+                <OptionButton Name="StationRecordsFilterType"
+                    MinWidth="200" Margin="0 0 10 0"/>
+                <LineEdit Name="StationRecordsFiltersValue"
+                    StyleClasses="actionSearchBox" HorizontalExpand="True"/>
+            </BoxContainer>
+            <BoxContainer Margin="0 0 0 5" HorizontalAlignment="Right" HorizontalExpand="true"
```

Comment:
did i fix it or not?

Reply:
no, i mean that all buttons should be on the same line: right now you have

```
[ Filter Select ][ Line Edit ]
           [ Search ][ Reset ]
```

what i was asking for was

```
[ Filter Select ][ Line Edit ][ Search ][ Reset ]

Reply:
it would considerably shrink the text edit line, so maybe it would be nicer to shrink the text inside the buttons/option selector too so it's more concise

Reply:
i mede this in some view
![image](https://user-images.githubusercontent.com/2144635/230068968-4292ffe4-3de7-4b67-b1fe-502a213c4c2d.png)